spec/03-plugins/31-proxy-cache/02-access_spec.lua

```diff
@@ -1346,5 +1361,34 @@ do
         assert.same("Bypass", res.headers["X-Cache-Status"])
       end)
     end)
+
+    it("ignore uri case in cache_key", function()
```

Comment:
We also need a test that proves the default behaviour is to be case sensitive, i.e. with `ignore_uri_case = false` the cache keys are different

Reply:
I've added another test (route-21), where I set `ignore_uri_case` to false (even if that is the default behaviour.